fieldservice/models/fsm_person.py

```diff
@@ -1,116 +1,139 @@
-# Copyright (C) 2018 - TODAY, Open Source Integrators
+# Copyright (C) 2020 Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
 
 
 class FSMPerson(models.Model):
-    _name = 'fsm.person'
-    _inherits = {'res.partner': 'partner_id'}
-    _description = 'Field Service Worker'
+    _name = "fsm.person"
+    _inherits = {"res.partner": "partner_id"}
+    _description = "Field Service Worker"
```

Comment:
I wish black reformatting was in a separate commit, so that we could have a diff with only the actual code changes needed for the migration...

Reply:
I agree. That would help for final code review of this one

Reply:
@dreispt @brian10048 Black reformatting is already done, Is it the way to change it?

Reply:
@murtuzasaleh The commit history would have to be modified

The idea is 
- black formatting in one commit
- other migration changes in separate commit

Because the black formatting makes a lot of minor changes, the separate commits would help us to see only the migration changes at one time